workflows/raredisease.nf

```diff
@@ -118,98 +122,136 @@ workflow RAREDISEASE {
+    ch_reduced_penetrance             = params.reduced_penetrance     ? Channel.fromPath(params.reduced_penetrance).collect()                                          : Channel.value([])
+    ch_score_config_snv               = params.score_config_snv       ? Channel.fromPath(params.score_config_snv).collect()                                            : Channel.value([])
+    ch_score_config_sv                = params.score_config_sv        ? Channel.fromPath(params.score_config_sv).collect()                                             : Channel.value([])
+    ch_target_bed_unprocessed         = params.target_bed             ? Channel.fromPath(params.target_bed).map{ it -> [[id:it[0].simpleName], it] }.collect()         : Channel.value([])
+    ch_variant_catalog                = params.variant_catalog        ? Channel.fromPath(params.variant_catalog).collect()                                             : Channel.value([])
+    ch_variant_consequences           = Channel.fromPath("$projectDir/assets/variant_consequences_v1.txt", checkIfExists: true).collect()
+    ch_vcfanno_resources_unprocessed  = params.vcfanno_resources      ? Channel.fromPath(params.vcfanno_resources).map{ it -> [[id:it[0].simpleName], it] }.collect()  : Channel.value([])
+    ch_vep_cache                      = params.vep_cache              ? Channel.fromPath(params.vep_cache).collect()                                                   : Channel.value([])
+    ch_vep_filters                    = params.vep_filters            ? Channel.fromPath(params.vep_filters).collect()                                                 : Channel.value([])
+
+    // Generate pedigree file
     MAKE_PED (CHECK_INPUT.out.samples.toList())
 
-    // STEP 0: QUALITY CHECK.
-    FASTQC (
-        CHECK_INPUT.out.reads
-    )
+    // Input QC
+    FASTQC (CHECK_INPUT.out.reads)
+    ch_versions = ch_versions.mix(CHECK_INPUT.out.versions)
     ch_versions = ch_versions.mix(FASTQC.out.versions.first())
 
-    // STEP 0: PREPARE GENOME REFERENCES AND INDICES.
+    // Prepare references and indices.
     PREPARE_REFERENCES (
-        params.aligner,
-        params.bwa_index,
-        params.bwamem2_index,
-        params.fasta,
+        ch_genome_fasta_no_meta,
+        ch_genome_fasta_meta,
         params.fasta_fai,
-        params.gnomad,
-        params.gnomad_af,
-        params.gnomad_af_tbi,
-        params.known_dbsnp,
-        params.known_dbsnp_tbi,
-        params.target_bed,
-        params.variant_catalog,
-        params.vcfanno_resources
+        ch_mt_fasta_shift_no_meta,
+        ch_mt_fasta_shift_meta,
+        ch_gnomad_af_tab,
+        ch_gnomad_vcf_unprocessed,
+        ch_known_dbsnp,
+        ch_target_bed_unprocessed,
+        ch_vcfanno_resources_unprocessed
     )
     .set { ch_references }
-    ch_versions = ch_versions.mix(ch_references.versions)
+
+    // Gather built indices or get them from the params
```

Comment:
Some lines here are very long. I would consider splitting them on two or more lines. I don't know if a standard has been agreed upon but python I think it's considered good practice to cap it at ~100 - 120 to improve readability. Even though they are very nicely aligned :)